app/Helpers/database/game.php

```diff
@@ -949,8 +950,12 @@ function modifyGameRichPresence(string $username, int $gameId, string $dataIn):
         return false;
     }
 
-    $game->RichPresencePatch = $dataIn;
-    $game->save();
```

Comment:
We still need to do this (or fetch the script from the triggers table).

After modifying the rich presence for a game (from the webpage), the change is written to the triggers table, but doesn't appear when refreshing the page, or opening the game from a client.

Reply:
Fixed in latest. I think in an earlier version of this branch (pre-push), I was playing with doing all the writes in the action and ultimately abandoned that idea but forgot to add this back.